examples/auth0/components/header.tsx

```diff
@@ -11,30 +11,30 @@ const Header = ({ user, loading }: HeaderProps) => {
       <nav>
         <ul>
           <li>
-            <Link href="/">
+            <Link href="/" legacyBehavior>
               <a>Home</a>
```

Comment:
Seems like a bug: i would expect this anchor to be removed

Reply:
Do we know why a style tag would affect the link? Seems like emitted HTML should be the same since “Link” becomes “a” with the new behavior.